File: Hayaa.ServiceFrame/ServiceFrameClient/Hayaa.ServicePlatform.Client/Config/ServicePlatformServiceConfig.cs
using Hayaa.BaseModel;
using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.ServicePlatform.Client.Config
{
    /// <summary>
    /// 服务配置类
    /// </summary>
    [Serializable]
    internal class ServicePlatformServiceConfig : BaseData, ConfigContent
    {
        /// <summary>
        /// App的组件
        /// </summary>
       public List<AppComponent> Components { set; get; }

        /// <summary>
        /// App功能验证收集与验证地址
        /// </summary>
        public String AppServiceUrl { set; get; }
        /// <summary>
        /// 心跳与
        /// </summary>
        public String HeartServiceUrl { set; get; }
        /// <summary>
        /// 心跳间隔，单位秒
        /// </summary>
        public int HeartTimespan { set; get; }
        /// <summary>
        /// App默认执行用户Id
        /// </summary>
        public int DefaultAppUserId { set; get; }
        public AppSettings AppSettings { set; get; }
        public ConnectionStrings ConnectionStrings { set; get; }
    }
}

File: Hayaa.ServiceFrame/ServiceFrameClient/hayaa.serviceplatform.client_java/hayaa.serviceplatform.client/src/main/java/hayaa/serviceplatform/client/ServicePlatformServiceConfig.java
package hayaa.serviceplatform.client;

import hayaa.basemodel.model.ConfigContent;

import java.io.Serializable;
import java.util.List;

class ServicePlatformServiceConfig extends ConfigContent implements Serializable {
    private List<AppComponent> Components;
    private String AppAuthServiceUrl;
    private String HeartServiceUrl;
    private int HeartTimespan;
    private int DefaultAppUserId;

    public List<AppComponent> getComponents() {
        return Components;
    }

    public void setComponents(List<AppComponent> components) {
        Components = components;
    }

    public String getAppAuthServiceUrl() {
        return AppAuthServiceUrl;
    }

    public void setAppAuthServiceUrl(String appAuthServiceUrl) {
        AppAuthServiceUrl = appAuthServiceUrl;
    }

    public String getHeartServiceUrl() {
        return HeartServiceUrl;
    }

    public void setHeartServiceUrl(String heartServiceUrl) {
        HeartServiceUrl = heartServiceUrl;
    }

    public int getHeartTimespan() {
        return HeartTimespan;
    }

    public void setHeartTimespan(int heartTimespan) {
        HeartTimespan = heartTimespan;
    }

    public int getDefaultAppUserId() {
        return DefaultAppUserId;
    }

    public void setDefaultAppUserId(int defaultAppUserId) {
        DefaultAppUserId = defaultAppUserId;
    }



}
